WeatherAPI2/Startup.cs

```diff
@@ -74,16 +74,13 @@ IEdmModel GetEdmModel()
 
         private static void SetOutputFormatters(IServiceCollection services)
         {
-            services.AddMvcCore(options =>
+            services.AddMvcCore(op =>
```

Comment:
Why did you shorten this out?

Reply:
There is no reason, I just copied from my application code, that is written in this way, I will change

Reply:
Fixed